src/main/webapp/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
I don't think you should name this file index.html if its just a tester file you'll rename later; index.html is reserved for " the most common name used for the default page shown on a website if no other page is specified when a visitor requests the site"